UNRELEASED.md

```diff
@@ -1,2 +1,4 @@
 # Unreleased
 
+- BPKCalendar:
```

Comment:
Can we include the change category(Added) here and use `Backpack/Calendar` instead of `BPKCalendar`.

```suggestion
**Added:**
- Backapck/Calendar
  - Exposed some properties of the calendar's underlying scrollView.
```

For completeness wouldn't it be nice to list all properties?